mach/proto/as/comm2.y

```diff
@@ -22,6 +22,7 @@ static item_t	*last_it, *o_it;
 %union {
 	word_t	y_word;
 	valu_t	y_valu;
+	int64_t	y_valu8;
```

Comment:
Is it worth having two different members here, or should we just define valu_t to an int64? (I can imagine problems if something writes a y_valu8 but reads a y_valu by mistake, which wouldn't be visible on little-endian machines.)

Reply:
valu_t gets redefined in mach/{arm, i386, i86, m68020, m68k2, mips, ns, powerpc, vax4, vc8, z8000}/as/mach0.c. Maybe we can remove those redefinitions and just let proto/as/comm0.h define valu_t as int64_t, but I haven't tried it yet.